### ISSUE TITLE
Build Errro : BUILD.bazel:2613:19: Compiling src/amalgam/gen/avx512amx.c failed: (Exit 1):

### ISSUE THREAD

**Issue by FunnyWii** [google/mediapipe#5345]
### OS Platform and Distribution

Ubuntu20.04

### Compiler version

g++9.4.0

### Programming Language and version

C++11

### Installed using virtualenv? pip? Conda?(if python)

NO

### MediaPipe version

0.10.13

### Bazel version

6.1.1

### Android AAR (if android)

None

### OpenCV version (if running on desktop)

4.7.0

### Describe the problem

When I run the cmd "bazel build --copt -DMESA_EGL_NO_X11_HEADERS --copt -DEGL_NO_X11 mediapipe/examples/desktop/face_detection:face_detection_gpu --local_ram_resources=1500  --local_cpu_resources=1", the following error happened

### Complete Logs

```shell
INFO: Build option --define has changed, discarding analysis cache.
WARNING: /home/funnywii/Documents/CV/PoseDet/mediapipe/mediapipe/framework/tool/BUILD:200:24: in cc_library rule //mediapipe/framework/tool:field_data_cc_proto: target '//mediapipe/framework/tool:field_data_cc_proto' depends on deprecated target '@com_google_protobuf//:cc_wkt_protos': Only for backward compatibility. Do not use.
WARNING: /home/funnywii/Documents/CV/PoseDet/mediapipe/mediapipe/framework/BUILD:69:24: in cc_library rule //mediapipe/framework:calculator_cc_proto: target '//mediapipe/framework:calculator_cc_proto' depends on deprecated target '@com_google_protobuf//:cc_wkt_protos': Only for backward compatibility. Do not use.
INFO: Analyzed target //mediapipe/examples/desktop/face_detection:face_detection_gpu (39 packages loaded, 6688 targets configured).
INFO: Found 1 target...
INFO: From Compiling mediapipe/framework/tool/message_type_util.cc [for tool]:
mediapipe/framework/tool/message_type_util.cc: In static member function 'static int mediapipe::DescriptorReader::MatchingPrefixLength(const string&, const string&)':
mediapipe/framework/tool/message_type_util.cc:67:14: warning: comparison of integer expressions of different signedness: 'int' and 'const long unsigned int' [-Wsign-compare]
   67 |     while (i < std::min(s.size(), t.size()) && s[i] == t[i]) {
      |            ~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
ERROR: /home/funnywii/.cache/bazel/_bazel_funnywii/4274885c1bcb543952ac607fcb912a97/external/XNNPACK/BUILD.bazel:2613:19: Compiling src/amalgam/gen/avx512amx.c failed: (Exit 1): gcc failed: error executing command (from target @XNNPACK//:avx512amx_prod_microkernels) /usr/bin/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer -MD -MF ... (remaining 41 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
gcc: error: unrecognized command line option '-mamx-int8'
Target //mediapipe/examples/desktop/face_detection:face_detection_gpu failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 67.686s, Critical Path: 65.02s
INFO: 252 processes: 138 internal, 114 linux-sandbox.
FAILED: Build did NOT complete successfully
```
```

**Comment by FunnyWii** [comment]
The error tells `ERROR: /home/funnywii/.cache/bazel/_bazel_funnywii/4274885c1bcb543952ac607fcb912a97/external/XNNPACK/BUILD.bazel:2613:19: Compiling src/amalgam/gen/avx512amx.c failed: (Exit 1):`

Could you please tell me how to resolve this problem?

**Comment by FunnyWii** [comment]
> The error tells `ERROR: /home/funnywii/.cache/bazel/_bazel_funnywii/4274885c1bcb543952ac607fcb912a97/external/XNNPACK/BUILD.bazel:2613:19: Compiling src/amalgam/gen/avx512amx.c failed: (Exit 1):`
> 
> Could you please tell me how to resolve this problem?

BTW, the "helloworld" can be built and run successfully.

**Comment by FunnyWii** [comment]
Solved by upgrading gcc to 10.5.0

Issue is closed
